code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -222,7 +222,7 @@
 	..()
 
 //Orange
-/datum/chemical_reaction/slime/slimecasp
+/datum/chemical_reaction/slime/slimecapsaicin
```

Comment:
is this the only place this typepath is mentioned?

Reply:
yes

Reply:
snake case https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#type-paths-must-be-snake-case